paramiko/auth_handler.py

```diff
@@ -372,12 +379,19 @@ def _parse_userauth_request(self, m):
         username = m.get_text()
         service = m.get_text()
         method = m.get_text()
-        self.transport._log(DEBUG, 'Auth request (type=%s) service=%s, username=%s' % (method, service, username))
+        self.transport._log(
+            DEBUG,
+            'Auth request (type=%s) service=%s, username=%s' % (
+                method, service, username))
         if service != 'ssh-connection':
             self._disconnect_service_not_available()
             return
-        if (self.auth_username is not None) and (self.auth_username != username):
-            self.transport._log(WARNING, 'Auth rejected because the client attempted to change username in mid-flight')
+        if ((self.auth_username is not None) and
+                (self.auth_username != username)):
+            self.transport._log(
+                WARNING,
+                'Auth rejected because the client attempted to change '
+                'username in mid-flight')
```

Comment:
In some projects it's allowed to exceed the line-length for log messages and the like, so that if you e.g. grep for something like "change username in mid-flight" after seeing it in a log line, you find it.